app/scripts/controllers/eventRegistrations.js

```diff
@@ -72,8 +72,15 @@ angular.module('confRegistrationWebApp')
         }
       });
     });
+
     //turn on visible blocks
     var visibleBlocks = localStorage.getItem('visibleBlocks:' + conference.id);
+    if(typeof visibleBlocks === 'undefined' || visibleBlocks === null){
+      //Initally display (true) or not display (false) columns
+      angular.forEach($scope.blocks, function(value){
+        value.visible = true;
```

Comment:
By default it should look like it does now.

The specific requirements:

**Unlocked Display Columns -** 
_Unlock "Started," "Completed" and "Email" display columns to allow admin to remove from display screen (leaving the "First Name," "Last Name," "Registrant Type" and "Paid" locked)._